### ISSUE TITLE
Endorser UI

### ISSUE THREAD

**Issue by esune** [hyperledger/aries-endorser-service#29]
The endorser service would benefit from a user interface allowing authorized users to manage endorsement requests.

The UI would facilitate assessing the type of request (public DID, schema, credential definition, revocation registry, etc.) and accepting/rejecting it - this is extremely useful in scenarios where teh `auto` flags are not set, or are set to only process a limited set of requests.

**Comment by esune** [comment]
@swcurran agreed. I think this issue would cover at **least** a basic UI that would allow an endorser service to be easily controlled by a human (so not relying only on the API).

As business requirements like the above are refined new features could be pushed out. I wasn't sure whether the business requirements would be part of this repo or be pushed to this repo from elsewhere - [this](https://github.com/bcgov/dts-endorser-service/issues/22) is a related, possibly instance-specific, issue that may end up generating new details for the UI as well.

**Comment by esune** [comment]
[This issue](https://github.com/bcgov/von/issues/378) has thoughts/ideas that we might want to consider when tackling this work.

**Comment by loneil** [comment]
Adding link to rapid-development POC herr https://github.com/loneil/endorser-ui
(note all the documentation in there is just the Tenant UI stuff, that needs updating)
